wdl2cwl/tests/test_cwl.py

```diff
@@ -18,11 +18,32 @@ def get_file(path: str) -> str:
         ),
     ],
 )
-def test_wdls(wdl_path: str, cwl_path: str) -> None:
-    """Test WDL to CWL conversion."""
-    convertedStr = wdl.main([get_file(wdl_path)])
-    testStr = ""
-    with open(get_file(cwl_path)) as file:
-        testStr = file.read()
-
-    assert convertedStr == testStr
+class TestParametrized:
+    """Contains the test functions for WDL to CWL conversion."""
+
+    def test_wdls(self, wdl_path: str, cwl_path: str) -> None:
+        """Test WDL to CWL conversion."""
+        convertedStr = wdl.convert(get_file(wdl_path))
+        testStr = ""
+        with open(get_file(cwl_path)) as file:
+            testStr = file.read()
+
+        assert convertedStr == testStr
+
+    def test_wdls_patch(
+        self, monkeypatch: pytest.MonkeyPatch, wdl_path: str, cwl_path: str
+    ) -> None:
+        """Test WDL to CWL conversion with patch."""
+        monkeypatch.setattr(
+            "sys.argv",
+            [
+                "python",
+                get_file(wdl_path),
+            ],
+        )
+        convertedStr = wdl.main()
```

Comment:
`main()` shouldn't be returning a string, can you please pass `--output` and include a temporary path using the `tmp_path` PyTest fixture https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture\

This will also improve your code coverage

 then compare that temp file to the `cwl_path` file?